pkg/testutil/rand/model.go

```diff
@@ -0,0 +1,155 @@
+package rand
+
+import (
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+type modelType interface {
+	model.Service | model.Message | model.Messages
+}
+
+// mdl returns a random modelType. It uses the given randF to generate the modelType.
+func mdl[T modelType, O ~func(*T)](randF func() *T, opts ...O) *T {
```

Comment:
This function name doesn't convey what it does.

Reply:
i was stuck with function name as model is reserved already by pkg 
if there is some better ideas, i am all ears :)

Reply:
mdl -> randomModelType ?
slice -> randomModelTypes ?

Reply:
i need to think about that